Cargo.toml

```diff
@@ -97,7 +97,7 @@ libp2p-quic = { version = "0.10.0", path = "transports/quic" }
 libp2p-relay = { version = "0.17.0", path = "protocols/relay" }
 libp2p-rendezvous = { version = "0.14.0", path = "protocols/rendezvous" }
 libp2p-request-response = { version = "0.26.0", path = "protocols/request-response" }
-libp2p-server = { version = "0.12.3", path = "misc/server" }
+libp2p-server = { version = "0.12.4", path = "misc/server" }
```

Comment:
I am surprised `libp2p-server` isn't bumped yet but I guess upgrading `libp2p` itself doesn't deserve a minor bump :)

Reply:
I don't feel strongly about this. I am not aware of a change in `libp2p` that would be visible at the surface of the binary. In other words, what did we change that changes the behavior of the binary?